pkg/skuba/actions/node/join/join.go

```diff
@@ -93,6 +92,8 @@ func Join(joinConfiguration deployments.JoinConfiguration, target *deployments.T
 		statesToApply = append([]string{"kubernetes.join.upload-secrets"}, statesToApply...)
 	}
 
+	statesToApply = append([]string{"kubeadm.reset"}, statesToApply...)
```

Comment:
Why remove the `fmt`?

Can't we just put the `prepend` in front and note remove the output - or is this also a problem with the new state ordering?

Reply:
@bergmannf I'vre already put it back, please don't ask why I removed this line ;)